libraries/bsg_manycore_packet.h

```diff
@@ -50,8 +50,8 @@ extern "C" {
         static void hb_mc_response_packet_fill(hb_mc_response_packet_t *rsp,
                                                const hb_mc_request_packet_t *req)
         {
-                hb_mc_response_packet_set_y_dst(rsp, hb_mc_request_packet_get_y_dst(req));
```

Comment:
I thought weren't having the host software respond to packets?

Reply:
It doesn't. This was a bug so I fixed it.

Reply:
Yeah but what is this function for to begin with?

Reply:
To create response packets.

Reply:
But why? When is it used?

Reply:
Now, it's used in the network simulator. It was also used in the host responses before I removed that

Reply:
So it doesn't have an impact on this pr?

Reply:
No.